langchain4j-redis/src/main/java/dev/langchain4j/store/embedding/redis/RedisSchema.java

```diff
@@ -34,7 +34,17 @@ class RedisSchema {
     private final int dimension;
     private final MetricType metricType;
 
-    RedisSchema(String indexName, String prefix, String vectorFieldName, String scalarFieldName, Collection<String> metadataKeys, VectorAlgorithm vectorAlgorithm, int dimension, MetricType metricType) {
+    RedisSchema(String indexName,
+                String prefix,
+                String vectorFieldName,
+                String scalarFieldName,
+                Collection<String> metadataKeys,
+                VectorAlgorithm vectorAlgorithm,
+                int dimension,
+                MetricType metricType) {
+        if (!prefix.endsWith(":")) {
+            prefix += ":";
```

Comment:
Should we do this? Documentation says:

> should be ended with ":" (e.g. embedding:)

So I would either remove or document this behavior

Reply:
Sounds reasonable. Currently I remove it and replace with check `prefix.endWith(":")` is true.